flags/tests/test_models.py

```diff
@@ -1,12 +1,41 @@
-from django.test import TestCase
+from django.core.cache import cache
+from django.test import TestCase, override_settings
 
 from flags.models import FlagState
 
 
 class FlagStateTestCase(TestCase):
+
     def test_flag_str(self):
-        state = FlagState.objects.create(name='MY_FLAG',
-                                         condition='boolean',
-                                         value='True')
-        self.assertEqual(str(state),
-                         'MY_FLAG is enabled when boolean is True')
+        state = FlagState.objects.create(
+            name='MY_FLAG',
+            condition='boolean',
+            value='True'
+        )
+        self.assertEqual(
+            str(state),
+            'MY_FLAG is enabled when boolean is True'
+        )
+
+    @override_settings(FLAGS_CACHE_CONDITIONS=True)
+    def test_save_signal_receiver(self):
+        cache.set('flags', ('test conditions', 'value', None))
+        self.assertIsNotNone(cache.get('flags'))
+        FlagState.objects.create(
+            name='MY_FLAG',
+            condition='boolean',
+            value='True'
+        )
+        self.assertIsNone(cache.get('flags_conditions_MY_FLAG'))
```

Comment:
Is this testing what you mean to test -- this cache key wouldn't be used any longer, would it?

Reply:
Oops, yes. I left in code from the other branch.